### PR TITLE
vlad.metelap

### PR DESCRIPTION
**Pull request by westnick**
Added  image.rb

### CODE REVIEW

File: vlad.metelap/image.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+require 'faraday'
+require "open-uri"
```
Comment (Flippylolz) [inline vlad.metelap/image.rb]: You don't need `open-uri` here. Please fix other rubocop issue so I can review the code

**Review by Flippylolz — CHANGES_REQUESTED** [review]

File: vlad.metelap/image.rb (hunk 1 above)
Comment (westnick) [reply vlad.metelap/image.rb]: corrected done.

File: vlad.metelap/homework_07/image.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    file_ext = string_headers.split('/').last
+    File.open("image.#{file_ext}", 'wb') { |fp| fp.write(response.body) }
+  rescue StandardError => e
+    puts e.message
+    puts e.backtrace.inspect
+  end
+end
+
+url = 'https://lostisland.github.io/faraday/assets/img/logo.png'
+obj = Image.download(url)
```
Comment (Flippylolz) [inline vlad.metelap/homework_07/image.rb]: Please fix the last rubocop warning so this request can be merged

Comment (slenderock) [inline vlad.metelap/homework_07/image.rb]:
```suggestion
Image.download(url)
```

**Review by Flippylolz — APPROVED** [review]

**Review by slenderock — APPROVED** [review]

**Pull request merged**